examples/arduino/LVGL_Arduino/LVGL_Arduino.ino

```diff
@@ -15,9 +15,10 @@
 //#include <examples/lv_examples.h>
 //#include <demos/lv_demos.h>
 
-/*Set to your screen resolution*/
+/*Set to your screen resolution and rotation*/
 #define TFT_HOR_RES   320
 #define TFT_VER_RES   240
```

Comment:
> For my display to work, I have to set
> 
> ```
> #define TFT_HOR_RES   240
> #define TFT_VER_RES   320
> 
>  ((lv_tft_espi_t *)(lv_display_get_driver_data(disp)))->tft->setRotation(0);
> ```

Based on what you said earlier, should these values be swapped in the example? Or is it correct the way it is?

```suggestion
#define TFT_HOR_RES   240
#define TFT_VER_RES   320
```

Reply:
I think this depends on the display. I would leave it as it is.

In TFT_eSPI, you can set the horizontal and vertical resolution, and the rotation. All this now can be set also in the "LVGL_Arduino.ino" example. For me this is perfect.